### DISCUSSION TITLE
Help please

### DISCUSSION THREAD

**Discussion by kooglMEbtch22** [community/community#62918] [Code Security]
### Select Topic Area

Question

### Body

phones and puts things that arnt real on there causing us to fight and try to make me kill myself. this man is a monster, he is a cyber cereal killer and im not exagerating he is trying to kill me. please help me.

07/20/2023, at my devices: Eric Veri remotly terrorizes my life and hes now gotten into my parents computers and taken over there phones and internet and he does everything threw my domains so it lookslike im the one doing it. He has stolen my life and destroyed my future and i desperatly need help. please
 Please I can only paste little things on here because he completely controls everything with aarch64 remote software and Android x and I can't stop him. [redacted due to PII]

**Comment by Yash-Dobariya** [comment]
**Contact the Authorities:** If you believe you are in immediate danger or facing cyber threats, contact your local law enforcement or emergency services right away. They are equipped to handle such situations and can provide appropriate assistance.

Seek Support: Reach out to friends, family, or someone you trust to let them know about your situation. They may be able to offer emotional support and help you in finding the right resources.

**Cybersecurity Expert:** Consider seeking help from a cybersecurity expert who can assess your devices and networks for any potential breaches or unauthorized access.

**Document Evidence:** If you suspect someone is hacking or using your devices illegally, try to gather evidence of the incidents. This may be helpful for authorities and cybersecurity experts to investigate the matter.

**Secure Your Devices**: Change passwords for all your accounts and devices, use two-factor authentication where possible, and ensure your software and operating systems are up to date.

**Online Counseling:** If you're feeling overwhelmed, consider seeking support from a mental health professional. Many online counseling platforms are available, and they can help you cope with stress and anxiety.

**Comment by queenofcorgis** [comment]
Hi @kooglMEbtch22 , Thanks for reaching out! 

We're sorry to hear you're experiencing this.

GitHub is a software hosting service, we host millions of projects, so probably every phone and computer on the planet has software on it that is also hosted here.

We're not able to provide support or answers for those individual projects. GitHub Support can only provide support for GitHub itself. As a result, I'm going to close and lock this discussion.

I'm sorry we can't help here but we'd recommend contacting a local IT specialist for assistance if you think your devices are affected by viruses or malware. They should be able to help you with these applications!
